<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeNotes</class>
 <widget class="QMainWindow" name="TreeNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TreeNotes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,5">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="indentation">
       <number>10</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>All Notes</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="titleEdit">
        <property name="placeholderText">
         <string>Ctrl + J</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="messageEdit">
        <property name="placeholderText">
         <string>Ctrl + H</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionSave"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSet_Icon"/>
    <addaction name="separator"/>
    <addaction name="actionSave_To_Disk"/>
    <addaction name="actionLoad_From_Disk"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionMove_Up"/>
    <addaction name="actionMove_Down"/>
    <addaction name="actionFocus_Parent"/>
    <addaction name="actionHide_Show_Note_Tree"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOther"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionSave"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSet_Icon"/>
   <addaction name="separator"/>
   <addaction name="actionMove_Up"/>
   <addaction name="actionMove_Down"/>
   <addaction name="actionFocus_Parent"/>
   <addaction name="actionHide_Show_Note_Tree"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionSet_Icon">
   <property name="text">
    <string>Set Icon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionFocus_Parent">
   <property name="text">
    <string>Focus Parent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionHide_Show_Note_Tree">
   <property name="text">
    <string>Hide/Show Note Tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSave_To_Disk">
   <property name="text">
    <string>Save To Disk</string>
   </property>
  </action>
  <action name="actionLoad_From_Disk">
   <property name="text">
    <string>Load From Disk</string>
   </property>
   <property name="toolTip">
    <string>Load From Disk</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
